com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -866,15 +869,59 @@ modules:
     sources: *dxvk_sources
 
 
+  - name: dxvk-nvapi
+    subdir: dxvk-nvapi
+    buildsystem: meson
+    build-options:
+      ldflags: ''
+      ldflags-override: true
+      config-opts:
+        - --cross-file=../build-win64.txt
+        - --cross-file=../meson-ccache-mingw64.txt
+        - --libdir=lib/wine/nvapi
+        - --bindir=lib/wine/nvapi
+      prepend-path: /usr/lib/sdk/mingw-w64/bin:/app/share/steam/compatibilitytools.d/Proton-Exp/bin
+    config-opts: &dxvk_nvapi_config_opts
+      - --buildtype=release
+      - --strip
+    post-install:
+      - install -Dm644 VERSION ${FLATPAK_DEST}/lib/wine/nvapi/version
```

Comment:
Just guessing: `post-install` is executed in the builddir in case of `builddir: true` (always true for meson), so we end up in `dxvk-nvapi/_flatpak_build`; thus, the path here should be `../VERSION`